src/redux/api.js

```diff
@@ -5,18 +5,20 @@ export default {
     createDictionary: data =>
       instance
         .post(`orgs/${data.owner}/collections/`, data)
-        .then(response => response.data)
+        .then(response => response)
         /* eslint-disable */
-        .then(() => { 
-          return instance.post(`orgs/${data.owner}/sources/`, data);
+        .then((res) => { 
+          instance.post(`orgs/${data.owner}/sources/`, data);
+          return res;
         }),
 
     createDictionaryUser: data =>
       instance
         .post('user/collections/', data)
-        .then(response => response.data)
-        .then(() => {
-          return instance.post('user/sources/', data);
+        .then(response => response)
```

Comment:
Must you chain promises when you are not processing them? You take in a response and return the same response.
E.g. In what you deleted, he takes in response and returns response.data and he uses the data to return something else. In your case, that line of code is not necessary.

Reply:
First you are wrong, as the `response` returned from these promises is being processed elsewhere, following the structure of the application. Secondly, the `data` being used is from the argument of the function and not the `response`. However, I agree with you that there might be no need  for the first `then` functions.

Reply:
I have the update.